libethereum/Client.h

```diff
@@ -348,8 +348,8 @@ class Client: public ClientBase, protected Worker
     Handler<h256 const&> m_tqReplaced;
     Handler<> m_bqReady;
 
-    bool m_wouldSeal = false;               ///< True if we /should/ be sealing.
-    bool m_wouldButShouldnot = false;       ///< True if the last time we called rejigSealing wouldSeal() was true but sealer's shouldSeal() was false.
+    std::atomic<bool> m_wouldSeal = { false };               ///< True if we /should/ be sealing.
+    std::atomic<bool> m_wouldButShouldnot = { false };       ///< True if the last time we called rejigSealing wouldSeal() was true but sealer's shouldSeal() was false.
```

Comment:
This is again not clear, it's touched only from one function in the single thread